repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/publication.ftl

```diff
@@ -50,49 +50,73 @@
     </div>
 </#macro>
 
+<@fmt.message key="headers.summary" var="summaryHeader" />
+<@fmt.message key="headers.related-links" var="relatedLinksHeader" />
+<@fmt.message key="headers.key-facts" var="keyFactsHeader" />
+<@fmt.message key="headers.administrative-sources" var="administrativeResourcesHeader" />
+<@fmt.message key="headers.datasets" var="datasetsHeader" />
+<@fmt.message key="headers.resources" var="resourcesHeader" />
+
+<#function getSectionNavLinks index>
+    <#assign links = [] />
+    
+    <#if index?has_content>
+        <#list index as i>
+            <#assign links += [{ "url": "#" + slugify(i), "title": i }] />
+        </#list>
+    </#if>    
+    
+    <#return links />
+</#function>
+
 <#macro fullContentOfPubliclyAvailablePublication>
     <@publicationHeader publication=publication/>
 
     <div class="grid-wrapper grid-wrapper--article <#if publication.pages?has_content>article-section</#if>" aria-label="Document Content">
         <div class="grid-row">
-            <@pageIndex index=index/>
+            <#if index?has_content && index?size gt 1>
+            <div class="column column--one-third page-block page-block--sidebar sticky sticky--top">
+                <@sectionNav getSectionNavLinks(index)></@sectionNav>
+            </div>
+            </#if>
 
             <div class="column column--two-thirds page-block page-block--main">
-                <div id="Summary" class="article-section article-section--summary no-border">
-                    <h2><@fmt.message key="headers.summary"/></h2>
+                <div id="${slugify(summaryHeader)}" class="article-section article-section--summary no-border">
```

Comment:
The IDs for these need to be kept in sync with the IDs in the index list. e.g uk.nhs.digital.ps.components.PublicationComponent#SUMMARY_ID.

I suggest you update the statics in PublicationComponent to be valid slugified values and then hard code the IDs in this template to the same values rather than using the user modifiable resource bundle values

Reply:
Makes sense, I'll do that